Nope, they're very related:

```
+ gmake check-whitespace
base/loading.jl:258:Use [`dirname`](@ref) to get the directory part and [`basename`](@ref) 
Error: trailing whitespace found in source file(s)
This can often be fixed with:
    git rebase --whitespace=fix HEAD~1
or
    git rebase --whitespace=fix master
and then a forced push of the correct branch
gmake: *** [Makefile:92: check-whitespace] Error 1
```

--- a/base/loading.jl
+++ b/base/loading.jl
@@ -255,7 +255,7 @@ locate_package(::Nothing) = nothing
 Return the path of `m.jl` file that was used to `import` module `m`,
 or `nothing` if `m` was not imported from a package.
 
-Use [`dirname`](@ref) to get the directory part and [`basename`](@ref) 
+Use [`dirname`](@ref) to get the directory part and [`basename`](@ref)
 to get the file name part of the path.
 """
 function pathof(m::Module)

